tests/test_visitors/test_ast/test_classes/test_instance_assignment.py

```diff
@@ -0,0 +1,28 @@
+from wemake_python_styleguide.violations.best_practices import (
+    InstanceLambdaAssignmentViolation,
+)
+from wemake_python_styleguide.visitors.ast.classes import (
+    InstanceAssignmentVisitor,
+)
+
+instance_lambda_assignment = """
+class Example(object):
+    def __init__(self):
+        self._attr = lambda: ...
```

Comment:
We need to make this `_attr` thing here parametrized.
We need to check: `attr`, `_attr`, and `__attr`

We also need to parametrize `self` part.
We need to check: `self`, `cls`, `mcs`, `other`

Reply:
Done